Jenkinsfile

```diff
@@ -95,47 +95,100 @@ pipeline {
       }
     }
 
-    stage('L2: BERT Squad v1.1') {
+    stage('L2: Parallel BERT SQUAD v1.1 / v2.0') {
       when {
         anyOf{
           branch 'candidate'
           changeRequest target: 'candidate'
         }
       }
       failFast true
-        steps {
-          sh 'cd examples/nlp && CUDA_VISIBLE_DEVICES=0 python question_answering_squad.py --precision 16 --amp_level=O1 --train_file /home/TestData/nlp/squad_mini/v1.1/train-v1.1.json --eval_file /home/TestData/nlp/squad_mini/v1.1/dev-v1.1.json --batch_size 8 --gpus 1 --do_lower_case --pretrained_model_name bert-base-uncased --optimizer adamw --lr 5e-5 --max_steps 2 --scheduler WarmupAnnealing'
-          sh 'rm -rf examples/nlp/lightning_logs && rm -rf /home/TestData/nlp/squad_mini/v1.1/*cache*'
+      parallel {
+        stage('BERT SQUAD 1.1') {
+          steps {
+            sh 'cd examples/nlp/question_answering && \
+            python question_answering_squad.py \
+            model.train_ds.file=/home/TestData/nlp/squad_mini/v1.1/train-v1.1.json \
+            model.validation_ds.file=/home/TestData/nlp/squad_mini/v1.1/dev-v1.1.json \
+            model.language_model.pretrained_model_name=bert-base-uncased \
+            model.version_2_with_negative=false \
+            pl.trainer.precision=16 \
+            pl.trainer.amp_level=O1 \
+            pl.trainer.gpus=[0] \
+            pl.trainer.max_steps=2 \
```

Comment:
instead of max_steps, let's do: "+pl.trainer.fast_dev_run=True" as it will test both train and validation steps

Reply:
Done.